src/Main.java

```diff
@@ -1,5 +1,90 @@
 public class Main {
+
+    int[] arr = generateRandomArray();
+
+    public static int[] generateRandomArray() {
+        java.util.Random random = new java.util.Random();
+        int[] arr = new int[30];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+
     public static void main(String[] args) {
-        System.out.println("Hello homework!");
+        lesson1();
+        System.out.println();
+        lesson2();
+        System.out.println();
+        lesson3();
+        System.out.println();
+        lesson4();
+    }
+
+
+    private static void lesson1() {
+        /*
+        Первым делом бухгалтеры попросили посчитать
+        сумму всех выплат за месяц.
+        */
+
+        int totalConsumptionMonth = 0;
+        for (int i = 0; i < generateRandomArray().length; i++) {
+            totalConsumptionMonth = totalConsumptionMonth + generateRandomArray()[i];
+        }
```

Comment:
for (int i = 0; i < arr.length; i++) {
            totalConsumptionMonth = totalConsumptionMonth + arr[i];
        }